sql/core/src/test/scala/org/apache/spark/sql/parquet/ParquetQuerySuite.scala

```diff
@@ -294,14 +294,14 @@ class ParquetQuerySuite extends QueryTest with FunSuiteLike with BeforeAndAfterA
     // Check to make sure that the attributes from either side of the join have unique expression
     // ids.
     query.queryExecution.analyzed.output.filter(_.name == "myint") match {
-      case Seq(i1, i2) if(i1.exprId == i2.exprId) =>
+      case Seq(i1, i2) if i1.exprId == i2.exprId =>
```

Comment:
Just a remind, `ParquetQuerySuite` is considered deprecated, and I'm going to remove it in #4116. I should have added a comment to make this clear. Didn't remove it at first because several Parquet PRs were out there and might cause unwanted merge conflicts.